automation/utilities/platform_utils.py

```diff
@@ -44,9 +45,14 @@ def get_firefox_binary_path():
                 "Current `FIREFOX_BINARY`: %s" % firefox_binary_path)
         return firefox_binary_path
 
-    root_dir = os.path.dirname(__file__)  # directory of this file
-    firefox_binary_path = os.path.abspath(root_dir +
-                                          "/../../firefox-bin/firefox-bin")
+    root_dir = os.path.dirname(__file__) + "/../.."
```

Comment:
Can we do `join()` here instead of `+`?

Reply:
I see we do `+` elsewhere, so I don't care too much about this. We don't support Windows anyway.